tests/lib/rules/no-container.test.ts

```diff
@@ -232,5 +322,101 @@ ruleTester.run(RULE_NAME, rule, {
 				},
 			],
 		},
+		{
+			code: `
+				const { container } = render(<Example />);
+				container.innerHTML;
+			`,
+			errors: [
+				{
+					line: 3,
+					column: 5,
+					messageId: 'noContainer',
+				},
+			],
+		},
+		{
+			code: `
+				const { container: alias } = render(<Example />);
+				alias.innerHTML;
+			`,
+			errors: [
+				{
+					line: 3,
+					column: 5,
+					messageId: 'noContainer',
+				},
+			],
+		},
+		{
+			code: `
+				const { container: { innerHTML } } = render(<Example />);
+			`,
+			errors: [
+				{
+					line: 2,
+					column: 11,
+					messageId: 'noContainer',
+				},
+			],
+		},
+		{
+			code: `
+				const { container: { innerHTML: alias } } = render(<Example />);
+			`,
+			errors: [
+				{
+					line: 2,
+					column: 11,
+					messageId: 'noContainer',
+				},
+			],
+		},
+		...SUPPORTED_TESTING_FRAMEWORKS.map(
+			(testingFramework) =>
+				({
+					settings: { 'testing-library/utils-module': 'test-utils' },
+					code: `
+        import { render } from '${testingFramework}'
+        const { container: { innerHTML } } = render(<Example />);
+      `,
+					errors: [
+						{
+							line: 3,
+							column: 15,
+							messageId: 'noContainer',
+						},
+					],
+				} as const)
+		),
+		{
+			settings: {
+				'testing-library/custom-renders': ['customRender', 'renderWithRedux'],
+			},
+			code: `
+        const { container } = renderWithRedux(<Example />);
+        container.innerHTML;
+      `,
+			errors: [
+				{
+					line: 3,
+					column: 9,
+					messageId: 'noContainer',
+				},
+			],
+		},
+		// {
+		// 	code: `
+		//     const view = render(<Example />);
+		//     view.container.innerHTML;
+		//   `,
+		// 	errors: [
+		// 		{
+		// 			line: 3,
+		// 			column: 29,
+		// 			messageId: 'noContainer',
+		// 		},
+		// 	],
+		// },
```

Comment:
@Belco90 This comment-out test is the test that is failing now, that shows us that we are not yet handling the `view.container.innerHTML` case

Reply:
I see. I need to go again through the utils for navigating the nodes, I think we had something to fix the problem described in the main description.